src/format/log/format-log-message.js

```diff
@@ -16,11 +17,17 @@ function formatLogMessage(contractName, eventName, message) {
             if (exc.constructor !== SyntaxError) throw exc;
             extraInfo = null;
           }
-          return assign({}, message, {
+          var formattedMessage = {
             extraInfo: extraInfo,
             marketCreationFee: speedomatic.unfix(message.marketCreationFee, "string"),
             topic: decodeTag(message.topic),
-          });
+          };
+          if (message.marketType === 1) {
```

Comment:
surprised we don't have a constant for this, perhaps i'll add it in a separate PR.

Reply:
Agree, that would be good!